test/type/_normalize.scss

```diff
@@ -6,25 +6,26 @@
 @use '../../../sass/tokens/api';
 @use '../../../sass/type/normalize';
 
-// Test Font Normalization
-// =======================
+$_global: config.$fonts;
+config.$fonts: data.$fonts;
```

Comment:
All these functions take a `$source` argument. If it's not set explicitly, they look up `config.$fonts`. We need to make sure they find the actual source map we're trying to test.

Reply:
Ah, ok. I saw you doing this in the test for `colors` and now that makes sense.